Umweltzone/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,185 @@
+        été publiées. Je vous demande donc personnellement de publier ces coordonnées <b>sous licence libre</b>.
+        Pour aider au traitement des données, les <b>formats suivants sont préférables</b> : ESRI Shapefiles,
+        CVS, GeoJSON.<br /> <br />
+        En vous remerciant de votre attention. <br /><br />
+        Cordialement </p>
+        <p> ----<br /><br />
+        Sehr geehrte Damen und Herren,<br /><br />         ich nutze die kostenfreie und quelloffene Anwendung &quot;Umweltzone&quot; [1],         mit der ich mich über den Verlauf von Umweltzonen und individuelle         Regelungen in verschiedenen Städten auf meinem Android-Gerät         informieren kann. Ich betrachte diese Anwendung als nützliche         Erweiterung zum bestehenden Informationsangebot vieler Städte.<br />         <br />         Leider kann die <b>Umweltzone %1$s</b> nicht dargestellt werden. Es fehlen         die Geokoordinaten des Umweltzonenumrings. Daher möchte ich Sie hiermit         persönlich bitten, die Koordinaten unter einer <b>freien Lizenz</b> zu         veröffentlichen. Um eine Weiterverarbeitung zu ermöglichen, eignen         sich folgenden <b>maschinenlesbare Dateiformate</b>:         ESRI Shapefiles, CSV, GeoJSON.<br />         <br />         Mit freundlichen Grüßen</p>          <p>___<br />         [1]: <a href="https://play.google.com/store/apps/details?id=de.avpptr.umweltzone">https://play.google.com/store/apps/details?id=de.avpptr.umweltzone</a></p>]]>     "</string>
+    <string name="trace_droid_dialog_title">"Envoyer un rapport de plantage &#8230;   "</string>
+    <string name="trace_droid_dialog_message">
+        "Un problème avec l&#8217;application Umweltzone a été détecté. Il serait très utile de nous envoyer
+        un rapport de plantage pour que nous puissions identifier la cause du problème."</string>
+    <string name="trace_droid_button_title_no">Non</string>
+    <string name="snack_engage_rate_title">Tout va bien ?</string>
+    <string name="snack_engage_rate_action">Merci de noter notre application</string>
+    <string name="snack_engage_beta_testing_title">Utilisateur avancé ?</string>
+    <string name="snack_engage_beta_testing_action">Devenez un beta-testeur</string>
+    <string name="settings_activity_title">Paramètres</string>
+    <string name="settings_category_title_data_privacy">Protection des données</string>
+    <string name="settings_title_google_analytics">Envoyer des statistiques d&#8217;usages anonymisées</string>
+    <string name="settings_summary_google_analytics">
+        "Autoriser la transmission de statistiques d&#8217;usage et des raports de plantages à Google Analytics.
+        Ces données anonymisées sont utilisées par les développeurs de Umweltzone pour analyser les erreurs
+        et bugs afin de continuer à améliorer l&#8217;application."</string>
+    <string name="settings_title_data_privacy_statement_de">"Manifeste de protection des données (en allemand)&#8230;  "</string>
```

Comment:
Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.

![screenshot](https://user-images.githubusercontent.com/144518/95884015-e5a70980-0d7b-11eb-8d7e-405d7b63a838.png)